### PR TITLE
update :es locale

### PR DESCRIPTION
**Pull request by acrogenesis**
I used https://www.localeapp.com/projects/8883 updated a lot of :es: translations, only left those that the translation would be unnecessary or wasn't sure how to translate it.

### CODE REVIEW

**Comment by farukaydin** [comment]
Thanks for PR. Maybe @dabit or other Spanish speakers wants to review it.

File: config/locales/es.yml [hunk 1]
```diff
@@ -1,178 +1,318 @@
...
+  doorkeeper:
+    applications:
+      buttons:
+        authorize: Autorizar
+        cancel: Cancelar
+        destroy: Destruir
+        edit: Editar
+        submit: Enviar
+      confirmations:
+        destroy: "¿Estas seguro?"
```
Comment (dabit) [inline config/locales/es.yml]: Estás

File: config/locales/es.yml [hunk 2]
```diff
@@ -1,178 +1,318 @@
...
+      form:
+        error: Ups! Checa la forma por posible errores
+      help:
+        native_redirect_uri: Usa %{native_redirect_uri} para pruebas locales
+        redirect_uri: Use una línea por URI
+        scopes:
+      index:
+        callback_url:
+        name: Nombre
+        new: Nuevo Aplicación
```
Comment (dabit) [inline config/locales/es.yml]: Nueva

File: config/locales/es.yml [hunk 3]
```diff
@@ -1,178 +1,318 @@
...
+        able_to: Esta aplicación podrá
+        prompt: "¿Autorizas a %{client_name} a usar tu cuenta?"
+        title: Autorización requerida
+      show:
+        title: Código de autorización
+    authorized_applications:
+      buttons:
+        revoke: Revocar
+      confirmations:
+        revoke: "¿Estas seguro?"
```
Comment (dabit) [inline config/locales/es.yml]: Estás

**Comment by dabit** [comment]
Other than small details LGTM :+1:

**Comment by acrogenesis** [comment]
Thanks @dabit! Done :+1:.

**Comment by arthurnn** [comment]
thanks

**Pull request merged**